src/steps/Microsoft.DotNet.UpgradeAssistant.Steps.Razor/IsExternalInit.cs

```diff
@@ -0,0 +1,12 @@
+﻿// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+
+namespace System.Runtime.CompilerServices
+{
+    /// <summary>
+    /// Workaround to enable init properties and record types while targeting netstandard2.0
+    /// </summary>
+    internal static class IsExternalInit
```

Comment:
If you merge from main, you can follow the pattern I introduced for the nullability attributes to include this file automatically if needed.

Reply:
Sounds good. Will do.